### ISSUE TITLE
While searching you need to enter the name of the city, (starting with a capital)

### ISSUE THREAD

**Issue by adridevlyon** [fransschreuder/libosmscout#18]
When using the https://open.uappexplorer.com/ version (0.4.1) on my Bq E4.5 with the whole france map downloaded, I am not able to search for a location (I have some suggestions after waiting several seconds, but not correct, as in the screen with an existing street, as underlined in the second screen), nor calculate any route, probably for the same reason

![screenshot20150627_113256526](https://cloud.githubusercontent.com/assets/12992234/8391404/3a386af2-1cc3-11e5-9638-267bcc7c3aaf.png)

![screenshot20150627_113217353](https://cloud.githubusercontent.com/assets/12992234/8391405/3f7f5494-1cc3-11e5-9cd1-e5f6c47a984b.png)

Maybe the size of the indexes is too big

Same problem when using the last official version (0.4.3 on official store), with only france/ile de france package

**Comment by fransschreuder** [comment]
Yes it is possible to do so, you only have to type the place as well while searching.
I know searching is a bit cumbersome and the suggestion box needs a lot of shaping but I am working on that.
See screenshots:
![screenshot20150627_132658396](https://cloud.githubusercontent.com/assets/218414/8391817/8aee0580-1cd0-11e5-8e03-a65a88e67e2f.png)
![screenshot20150627_132757878](https://cloud.githubusercontent.com/assets/218414/8391819/8f5d6bb0-1cd0-11e5-81f9-835d404d3134.png)

**Comment by adridevlyon** [comment]
Ok for the way to search it which is not very intuitive, but this isn't the same Rue d'Alembert in our two examples : I was seraching the one located at Issy les Moulineaux, not Paris.
How can I find what search to type when just knowing the city and the street name ?

**Comment by adridevlyon** [comment]
To improve the searching, maybe you can see with the online gps navigation app (https://launchpad.net/unav) ?

**Comment by fransschreuder** [comment]
How should the app know you want to go to Issy les Moulineaux if I don't even know? Just type the street name and the city as I did in the screenshot above. I typed Paris so I got Paris. If you type Issy, you get Issy les Moulineaux:
![screenshot20150627_141635449](https://cloud.githubusercontent.com/assets/218414/8391934/67941e7e-1cd7-11e5-8843-b85e10607638.png)

Your suggestion to look at unav is not very helpful, as it is online and not done from within the app. I am working on the search issue, and I know it sucks a little... you don't need to report on this, unless you have bright ideas on how to improve it.
